cli/lib/kontena/plugin_manager.rb

```diff
@@ -174,10 +174,9 @@ def load_plugins
                 $stderr.puts " [#{Kontena.pastel.red('error')}] Plugin #{Kontena.pastel.cyan(plugin_name)} (#{spec.version}) is not compatible with the current cli version."
                 $stderr.puts "         To update the plugin, run 'kontena plugin install #{plugin_name}'"
               end
-            rescue LoadError => ex
-              $stderr.puts " [#{Kontena.pastel.red('error')}] Failed to load plugin: #{spec.name}"
+            rescue LoadError, StandardError => ex
```

Comment:
rescue [`ScriptError`](https://ruby-doc.org/core-2.3.1/ScriptError.html) instead of `LoadError` in order to also handle `SyntaxError` and `NotImplementedError`?